includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -207,7 +208,8 @@ public function sendTemporaryPasswordEmail() {
 	/**
 	 * Api endpoint to send a password reset e-mail.
 	 * @requestParam user_id : \User::id
-	 * @requestParam token : user identification token
+	 * @requestParam reset_token : user identification token
+	 * @requestParam token_ctx : token context identifying the email content.
 	 */
 	public function sendPasswordResetLinkEmail() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)